pulsar-client-tools/src/main/java/org/apache/pulsar/shell/PulsarShell.java

```diff
@@ -414,18 +441,36 @@ public List<String> readCommand() {
                 return;
             }
             execState = ExecState.RUNNING;
-            final String line = words.stream().collect(Collectors.joining(" "));
-            if (StringUtils.isBlank(line)) {
+            final String inputLine = words.stream().collect(Collectors.joining(" "));
```

Comment:
perhaps `final` could be dropped so that the a single variable `line` could be used.

Reply:
Updated it